src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -62,15 +67,14 @@ const ExternalCourseEnrollment = () => {
     }
   }, [externalCourseFormSubmissionError, containerRef]);
 
-  const handleCheckoutSuccess = () => {
-    history.push('enroll/complete');
-  };
-
   useEffect(() => {
+    // Once a redemption has successfully completed and the can-redeem query has been invalidated or
+    // a user attempts to navigate directly to :slug/executive-education-2u/course/:courseKey/enroll,
+    //  it will run this conditional and perform the redirect
     if (hasSuccessfulRedemption) {
-      history.push('enroll/complete');
+      history.push({ pathname: completeEnrollmentUrl });
```

Comment:
nit: would passing `completeEnrollmentUrl` as a string argument versus within the object be sufficient (`history.push(completeEnrollmentUrl)`)?